UI/blog.html

```diff
@@ -96,24 +96,57 @@ <h3>FEATURES</h3><br/>
             </div>
             <div class="top">
                 <div class="top-subscribe">
-                <form>
+                <form id="subform">
                 <h3>SUBSCRIBE TO NEWSLETTER</h3>
                 <div class="form-box">
-                <input type="email" class="subscribe" placeholder="Enter Email">
-                <button class="sub-btn" type="submit">SUBSCRIBE</button>
+                <input type="text" class="subscribe" id="email" name="mail" placeholder="Enter Email">
+                <h5 id="h">error_message</h5>
+                <button id="submit" class="sub-btn" type="submit">SUBSCRIBE</button>
                 </div>
                 </form>
                 </div>
                 <p><br/><br/><br/>FOLLOW US </p>
-                <a href="https://web.facebook.com/mucyo.aimechrischristian"><i class="fa fa-facebook"></i></a>
-                <a href="https://twitter.com/chrisdolla17"><i class="fa fa-twitter"></i></a>
+                <a href="https://web.facebook.com/mucyo.aimechristian"><i class="fa fa-facebook"></i></a>
+                <a href="https://www.twitter.com/chrisdolla17"><i class="fa fa-twitter"></i></a>
                 <a href="https://www.instagram.com/chris_.dolla"><i class="fa fa-instagram"></i></a>
             </div>
         </div>
         <div class="footer2"><span>Copyright ©2020 All rights reserved</span>| This template is made by Mr Chris</div>
     </footer>
 </main>
 </div>
+<!--validating and storing subscribers-->
+<script>
```

Comment:
I was wondering if you can use a separate file for javascript and you connect to it.